manifests/enterprise/repo/apt.pp

```diff
@@ -0,0 +1,29 @@
+class sensu::enterprise::repo::apt {
```

Comment:
On an EL-based system, you actually need three repos installed:

- sensu (OSS),
- sensu-enterprise, and
- sensu-enterprise-dashboard